diff --git a/vendor/github.com/docker/docker/pkg/mount/flags.go b/vendor/github.com/docker/docker/pkg/mount/flags.go
index 607dbed..11c92f6 100644
--- a/vendor/github.com/docker/docker/pkg/mount/flags.go
+++ b/vendor/github.com/docker/docker/pkg/mount/flags.go
@@ -147,3 +147,7 @@ func ParseTmpfsOptions(options string) (int, string, error) {
 	}
 	return flags, data, nil
 }
+
+func ParseOptions(options string) (int, string) {
+	return parseOptions(options)
+}
