diff --git a/vendor/github.com/docker/docker/pkg/mount/flags.go b/vendor/github.com/docker/docker/pkg/mount/flags.go
index d2fb1fb..0f6fd61 100644
--- a/vendor/github.com/docker/docker/pkg/mount/flags.go
+++ b/vendor/github.com/docker/docker/pkg/mount/flags.go
@@ -90,3 +90,7 @@ func ParseTmpfsOptions(options string) (int, string, error) {
 	}
 	return flags, data, nil
 }
+
+func ParseOptions(options string) (int, string) {
+	return parseOptions(options)
+}
